docs/sql_reference/src/asciidoc/_chapters/sql_statements.adoc

```diff
@@ -5860,7 +5863,7 @@ column-list is:
 <<<
 === syntax description of grant
 
-* `_privilege_ [,_privilege_ ] &#8230; | all [privileges]`
+* `_privilege-name_ [,_privilege-name_ ] &#8230; | all [privileges]`
 +

```

Comment:
what is &#8230?

Reply:
&#8230; is the unicode for "…" (hellip) :smiley_cat:
For more information, see https://www.w3schools.com/charsets/ref_html_entities_h.asp.